seed/views/v3/properties.py

```diff
@@ -57,12 +59,91 @@
 ErrorState = namedtuple('ErrorState', ['status_code', 'message'])
 
 
-class PropertyViewSet(viewsets.ViewSet, OrgMixin, ProfileIdMixin):
+class PropertyViewFilterBackend(filters.DjangoFilterBackend):
+    """
+    Used to add filters to the `search` view
+    I was unable to find a better way to add the filterset_class to a single view
+
+    TODO: Add this to seed/filtersets.py or seed/filters.py
+    """
+    def get_filterset_class(self, view, queryset=None):
+        return PropertyViewFilterSet
+
+
+class PropertyViewFilterSet(filters.FilterSet, OrgMixin):
+    """
+    Advanced filtering for PropertyView sets
+
+    TODO: Add this to seed/filtersets.py
```

Comment:
Not moving immediately to avoid collision with the existing PropertyViewFilterSet in seed/filtersets.py